[2025-08-03T10:47:01.739+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:46:00+00:00/task_id=fetch_training_data permission to 509
[2025-08-03T10:47:01.741+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:46:00+00:00/task_id=fetch_training_data permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:46:00+00:00/task_id=fetch_training_data'
[2025-08-03T10:47:01.778+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:46:00+00:00/task_id=fetch_training_data permission to 509
[2025-08-03T10:47:01.779+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:46:00+00:00/task_id=fetch_training_data permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:46:00+00:00/task_id=fetch_training_data'
[2025-08-03T10:47:01.820+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: train_predict_to_db.fetch_training_data scheduled__2025-08-03T10:46:00+00:00 [queued]>
[2025-08-03T10:47:01.832+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: train_predict_to_db.fetch_training_data scheduled__2025-08-03T10:46:00+00:00 [queued]>
[2025-08-03T10:47:01.833+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-08-03T10:47:01.849+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): fetch_training_data> on 2025-08-03 10:46:00+00:00
[2025-08-03T10:47:01.860+0000] {standard_task_runner.py:57} INFO - Started process 830 to run task
[2025-08-03T10:47:01.864+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'train_predict_to_db', 'fetch_training_data', 'scheduled__2025-08-03T10:46:00+00:00', '--job-id', '70', '--raw', '--subdir', 'DAGS_FOLDER/train_predict_to_db_dag.py', '--cfg-path', '/tmp/tmpeckcsn4e']
[2025-08-03T10:47:01.868+0000] {standard_task_runner.py:85} INFO - Job 70: Subtask fetch_training_data
[2025-08-03T10:47:01.936+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:46:00+00:00/task_id=fetch_training_data permission to 509
[2025-08-03T10:47:01.937+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:46:00+00:00/task_id=fetch_training_data permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:46:00+00:00/task_id=fetch_training_data'
[2025-08-03T10:47:01.939+0000] {task_command.py:416} INFO - Running <TaskInstance: train_predict_to_db.fetch_training_data scheduled__2025-08-03T10:46:00+00:00 [running]> on host 407839faddcf
[2025-08-03T10:47:02.045+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='train_predict_to_db' AIRFLOW_CTX_TASK_ID='fetch_training_data' AIRFLOW_CTX_EXECUTION_DATE='2025-08-03T10:46:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-08-03T10:46:00+00:00'
[2025-08-03T10:47:02.061+0000] {db_utils.py:17} INFO - Connected to PostgreSQL
[2025-08-03T10:47:03.049+0000] {train_predict_to_db_dag.py:29} INFO - Fetched training data tail:
            id  store  ...  productname                 created_at
100612  156052    110  ...    product_A 2025-08-03 10:46:18.108804
100613  156053     70  ...    product_A 2025-08-03 10:46:27.899260
100614  156054    111  ...    product_A 2025-08-03 10:46:37.914782
100615  156055    113  ...    product_A 2025-08-03 10:46:47.924487
100616  156056    114  ...    product_A 2025-08-03 10:46:57.748672

[5 rows x 12 columns]
[2025-08-03T10:47:03.082+0000] {python.py:194} INFO - Done. Returned value was: {'store': 370, 'start_date': '2025-05-05', 'end_date': '2025-08-03'}
[2025-08-03T10:47:03.119+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=train_predict_to_db, task_id=fetch_training_data, execution_date=20250803T104600, start_date=20250803T104701, end_date=20250803T104703
[2025-08-03T10:47:03.206+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-08-03T10:47:03.241+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
